.circleci/config.yml

```diff
@@ -49,8 +49,8 @@ test-firefox: &test-firefox
         command: |
           # we should pin a version here, but llvm upstream now emits bulk memory for pthreads
           # which requires very latest nightly as of Jul 13 2019
-          wget -O nightly.tar.bz2 "https://download.mozilla.org/?product=firefox-nightly-latest-ssl&os=linux64&lang=en-US"
-          tar xf nightly.tar.bz2
```

Comment:
Do we really want nightly?  Seems like we want to pin to something with less churn if possible?

Reply:
(I think you meant beta? This moves to beta, not nightly.)

Pinning does make sense in a way, but it also has a downside - we were pinned to a quite old firefox before I upgraded it recently, we just forgot basically.

The least churn is to use stable versions of all browsers I guess. But we do need beta atm, for firefox due to passive segments and for chrome due to pthreads leaks.

Reply:
Sorry I read the diff backwards.